tasks/ts.ts

```diff
@@ -17,19 +18,60 @@ export = function(grunt: IGrunt) {
 			exclude: ['tests/**/*.ts', 'src/*/tests/**/*.ts', 'src/*/example/**/*.ts'],
 			compilerOptions: {
 				target: 'es6',
-				module: 'es6',
-				sourceMap: false,
+				module: 'esnext',
+				sourceMap: true,
 				outDir: 'dist/esm',
-				inlineSourceMap: true,
 				inlineSources: true
 			}
 		}
 	};
+	const postTasks: any = {
+		esm: [
+			function () {
+				grunt.task.registerTask('rename-mjs', () => {
+					// rename .js files to .mjs files
+					grunt.file.expand(['dist/esm/**/*.js'])
+						.forEach(file => fs.renameSync(file, file.replace(/\.js$/g, '.mjs')));
+
+					// rename .js.map files to .mjs.map files
```

Comment:
This will need to be renamed in the source map comment in the `.mjs` file as well

Reply:
Renamed!